test/base/SpecBase.scala

```diff
@@ -129,6 +129,48 @@ trait SpecBase extends PlaySpec with GuiceOneAppPerSuite with TryValues {
     .set(FREResponse, FRENoYears).success.value
     .set(FREAmounts, Seq(FlatRateExpenseAmounts(Some(FlatRateExpense(100)), TaiTaxYear()))).success.value
 
+  val currentYearMinus1UserAnswers = emptyUserAnswers
+    .set(EmployerContributionPage,  EmployerContribution.NoEmployerContribution).success.value
+    .set(TaxYearSelectionPage, Seq(CurrentYearMinus1)).success.value
+    .set(YourAddressPage, true).success.value
+    .set(YourEmployerPage, true).success.value
+    .set(ClaimAmount, 100).success.value
+    .set(ClaimAmountAndAnyDeductions, 80).success.value
+    .set(FREResponse, FRENoYears).success.value
+    .set(FREAmounts, Seq(FlatRateExpenseAmounts(Some(FlatRateExpense(100)), TaiTaxYear()))).success.value
+
+  def previousYearUserAnswers(years: Seq[TaxYearSelection]) = emptyUserAnswers
+    .set(EmployerContributionPage,  EmployerContribution.NoEmployerContribution).success.value
+    .set(TaxYearSelectionPage, years).success.value
+    .set(YourAddressPage, true).success.value
+    .set(YourEmployerPage, true).success.value
+    .set(ClaimAmount, 100).success.value
+    .set(ClaimAmountAndAnyDeductions, 80).success.value
+    .set(FREResponse, FRENoYears).success.value
+    .set(FREAmounts, Seq(FlatRateExpenseAmounts(Some(FlatRateExpense(100)), TaiTaxYear()))).success.value
+
+
+  val currentYearAndCurrentYearMinus1UserAnswers = emptyUserAnswers
+    .set(EmployerContributionPage,  EmployerContribution.NoEmployerContribution).success.value
+    .set(TaxYearSelectionPage, Seq(CurrentYear, CurrentYearMinus1)).success.value
+    .set(YourAddressPage, true).success.value
+    .set(YourEmployerPage, true).success.value
+    .set(ClaimAmount, 100).success.value
+    .set(ClaimAmountAndAnyDeductions, 80).success.value
+    .set(FREResponse, FRENoYears).success.value
+    .set(FREAmounts, Seq(FlatRateExpenseAmounts(Some(FlatRateExpense(100)), TaiTaxYear()))).success.value
+
+  val currentYearAndCurrentYearMinus2UserAnswers = emptyUserAnswers
+    .set(EmployerContributionPage,  EmployerContribution.NoEmployerContribution).success.value
+    .set(TaxYearSelectionPage, Seq(CurrentYear, CurrentYearMinus2)).success.value
+    .set(YourAddressPage, true).success.value
+    .set(YourEmployerPage, true).success.value
+    .set(ClaimAmount, 100).success.value
+    .set(ClaimAmountAndAnyDeductions, 80).success.value
+    .set(FREResponse, FRENoYears).success.value
+    .set(FREAmounts, Seq(FlatRateExpenseAmounts(Some(FlatRateExpense(100)), TaiTaxYear()))).success.value
+
```

Comment:
Could these not be made into a function that takes in a taxYearSelection if they are the same e.g.

def createUserAnswers(taxYearSelection: Seq[TaxYearSelection])) = emptyUserAnswers
.set(TaxYearSelectionPage, taxYearSelection).success.value
...
...
...